Makefile

```diff
@@ -202,6 +202,16 @@ THIRD_PARTY_NOTICE_LICENSE_TYPE = microsoft-oss
 $(eval $(call CREATE_THIRD_PARTY_NOTICES_RULE,ThirdPartyNotices.txt,foundation,False,False))
 $(eval $(call CREATE_THIRD_PARTY_NOTICES_RULE,bin/$(CONFIGURATION)/lib/xamarin.android/ThirdPartyNotices.txt,$(THIRD_PARTY_NOTICE_LICENSE_TYPE),True,False))
 
+MONODROID_PATH=$(topdir)/monodroid
```

Comment:
I would prefer that `MONODROID_PATH` be `$(topdir)/external/monodroid`.  Let's try *not* to pollute `$(topdir)`. :-)